docs/resources/Guild.md

```diff
@@ -460,3 +475,127 @@ Returns the [guild embed](#DOCS_GUILD/guild-embed-object) object. Requires the '
 ## Modify Guild Embed % PATCH /guilds/{guild.id#DOCS_GUILD/guild-object}/embed
 
 Modify a [guild embed](#DOCS_GUILD/guild-embed-object) object for the guild. All attributes may be passed in with JSON and modified. Requires the 'MANAGE_GUILD' permission. Returns the updated [guild embed](#DOCS_GUILD/guild-embed-object) object.
+
+## Get Guild Audit Log % GET /guilds/{guild.id#DOCS_GUILD/guild-object}/audit-log
+
+Returns an an [audit log object](#DOCS_GUILD/audit-log-object) for the guild. Requires the 'VIEW_AUDIT_LOG' permission.
+
+###### Query String Parameters
+
+|Field | Type | Description |
+| ---- | ----- | ---------- |
+| user_id | snowflake | filter the log for a user id |
+| action_type | int  | the type of [audit log event](#DOCS_GUILD/audit-log-events)|
+| before | snowflake | filter the log before a certain action id |
+| limit | int | how many entries are returned (default 50, minimum 1, maximum 100)
+
+### Audit Log Object
+
+###### Audit Log Object Structure
+
+| Field | Type | Description |
+|-------|------|-------------|
+| webhooks | array of [webhook object](#DOCS_WEBHOOK/webhook-object) | List of webhooks found in the audit log |
+| users | array of [user object](#DOCS_USER/user-object) | List of users found in the audit log |
+| audit_log_entries | array of [audit log entry object](#DOCS_GUILD/audit-log-entry-object) | List of audit log entires
+
+### Audit Log Entry Object
+
+###### Audit Log Entry Structure
+
+| Field | Type | Description |
+|-------|------|-------------|
+| target_id | string | id of the affected object (webhook, user, role, etc.) |
+| changes | array of [audit log change object](#DOCS_GUILD/audit-log-change-object) | changes made to the target id|
+| user_id | snowflake | the user who made the changes
+| id | snowflake | id of the entry
+| action_type | an [audit log event](#DOCS_GUILD/audit-log-event) | type of action that occured
+| options | object | additional information specific to the action type
```

Comment:
Not sure how you would expand on this but the `options` key is used in the following places:

1. `MEMBER_PRUNE` has a `delete_members_days` key as a `string` value with how far the prune went and a `members_removed` key (again with a `string` value) with how many members were pruned.
2. `MESSAGE_DELETE` has two keys, `channel_id` snowflake for where the message got deleted and `count` string for how many messages got deleted.
3. `CHANNEL_OVERWRITE_[CREATE|UPDATE|DELETE]` have an `id` snowflake with the overwrite that got changed and a `type` that says the overwrite type (which can be `member` or `role`). If the `type` is equal to `role` then there is another key, `role_name` which obviously has the role name.